### PR TITLE
[TM-1062,1063] alert edit polygon attributes

### PR DESCRIPTION
**Pull request by egrojMonroy**
Add alerts for edit geometry, save geometries in Site and Admin views.

### CODE REVIEW

File: src/components/elements/MapPolygonPanel/AttributeInformation.tsx [hunk 1]
```diff
@@ -156,11 +174,22 @@ const AttributeInformation = () => {
...
+            pathParams: { uuid: polygonData.uuid }
+          },
+          {
+            onSuccess: () => {
+              setShouldRefetchPolygonData(true);
+              setEditPolygon({ isOpen: false, uuid: "" });
+              displayNotification(t("Polygon data updated successfully"), "success", "Success!");
+            },
+            onError: error => {
+              displayNotification(t("Error updating polygon data"), "error", "Error!");
```
Comment (roguenet) [inline src/components/elements/MapPolygonPanel/AttributeInformation.tsx:189]: `Success!` on line 186 and `Error!` here need translation

File: src/context/mapArea.provider.tsx [hunk 2]
```diff
@@ -35,7 +47,14 @@ const defaultValue: MapAreaType = {
   shouldRefetchPolygonData: false,
   setShouldRefetchPolygonData: () => {},
   shouldRefetchValidation: false,
-  setShouldRefetchValidation: () => {}
+  setShouldRefetchValidation: () => {},
+  polygonNotificationStatus: {
+    open: false,
+    message: "",
+    type: "success",
+    title: "Success!"
```
Comment (roguenet) [inline src/context/mapArea.provider.tsx:55]: Needs translation

File: src/context/mapArea.provider.tsx [hunk 3]
```diff
@@ -51,7 +70,17 @@ export const MapAreaProvider: React.FC<{ children: ReactNode }> = ({ children })
...
+  const [polygonNotificationStatus, setpolygonNotificationStatus] = useState<{
+    open: boolean;
+    message: string;
+    type: "success" | "error" | "warning";
+    title: string;
+  }>({
+    open: false,
+    message: "",
+    type: "success",
+    title: "Success!"
```
Comment (roguenet) [inline src/context/mapArea.provider.tsx:82]: Needs translation

File: src/components/elements/MapPolygonPanel/AttributeInformation.tsx (hunk 1 above)
Comment (egrojMonroy) [reply src/components/elements/MapPolygonPanel/AttributeInformation.tsx:189]: done

File: src/context/mapArea.provider.tsx (hunk 3 above)
Comment (egrojMonroy) [reply src/context/mapArea.provider.tsx:82]: removed because is init state and shouldnt have title.

File: src/context/mapArea.provider.tsx (hunk 2 above)
Comment (egrojMonroy) [reply src/context/mapArea.provider.tsx]: removed, unnecesary text

**Review by roguenet — APPROVED** [review]

**Pull request merged**